modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -190,89 +167,38 @@ const makeRtcAdaptor = (tinymceEditor: Editor, rtcEditor: RtcRuntimeApi): RtcAda
     undoManager: {
       beforeChange: ignore,
       addUndoLevel: unsupported,
-      undo: () => {
-        rtcEditor.undo();
-        return createDummyUndoLevel();
-      },
-      redo: () => {
-        rtcEditor.redo();
-        return createDummyUndoLevel();
-      },
-      clear: () => rtcEditor.clear(),
-      reset: () => rtcEditor.reset(),
-      hasUndo: () => rtcEditor.hasUndo(),
-      hasRedo: () => rtcEditor.hasRedo(),
-      transact: (_undoManager, _locks, fn) => {
-        rtcEditor.transact(fn);
-        return createDummyUndoLevel();
-      },
-      ignore: (_locks, callback) => rtcEditor.ignore(callback),
-      extra: (_undoManager, _index, callback1, callback2) => rtcEditor.extra(callback1, callback2)
+      undo: () => rtcEditor.undoManager.undo(),
```

Comment:
There's a lot of duplication here that won't get minified, so should we looking at breaking that out? eg:
```
const undoManger = rtcEditor.undoManager;
const formatter = rtcEditor.formatter;
const editor = rtcEditor.editor;
....
```

Reply:
I've been wondering whether we should just implement the entire undoManager etc api in RTC. I'm not sure what's left that we aren't overriding.

Reply:
There are some minor things like a data array etc. But I prefer us having the same format on these overrides it is also now very easy to read. In theory we could have some form of make undoManager thing in the rtc that returns a whole object and just replaces the instance or something but I think that would be a bit more messy and not as clear as this.